mix.exs

```diff
@@ -17,6 +17,7 @@ defmodule Brcpfcnpj.Mixfile do
   defp deps do
     [{:earmark, "~> 0.2.1", only: :dev},
      {:ex_doc, "~> 0.11.4", only: :dev},
+     {:ecto, ">= 2.0.0"},
```

Comment:
I believe you forgot remove this line, am I correct ?
Not everyone uses project with ecto.